cocotb/share/makefiles/simulators/Makefile.nvc

```diff
@@ -0,0 +1,142 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (C) 2023 Sean Anderson <seanga2@gmail.com>
+# Copyright (c) 2014 Potential Ventures Ltd
```

Comment:
Yes, if you copied a non-trivial amount of code from another file keep the copyright line. But we do not add individual copyright lines for new code, hence the full header would be:

```
# Copyright cocotb contributors
# Copyright (c) 2014 Potential Ventures Ltd
# Licensed under the Revised BSD License, see LICENSE for details.
# SPDX-License-Identifier: BSD-3-Clause
```

Why don't we add individual copyright lines? The much better place for metadata about authorship is the git log; having similar information in the header isn't scalable. We're not alone in this approach, just look at [a random file in the Chromium repository](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/content/common/color_parser.cc#1), for example.

Reply:
I'm very sorry, but we do not add individual contributors to the license header for new code any more. This is one of our "house rules" written down in the contributor guide and a requirement to accepting code changes. We are happy to acknowledge your contributions (which we're very grateful about!) in other ways and do so whenever we can, but the file header is just not the right place for it.

Reply:
Perhaps we need to add a statement to the contributing documentation that states that all contributors must relinquish copyrights to the maintainers or FOSSi. But @Forty-Bot, if that wasn't explicit, it should be clear now. Any code contributed to the repo requires you to relinquish copyrights on the contributed code to the currently amorphous "us".

Reply:
I guess I should let @imphil chime in to what exactly the idea was here. "cocotb contributors" does imply more than the maintainers or FOSSi. Technically you are a "cocotb contributor", but at the same time "cocotb contributors" is not a legal entity that can possess copyright. It implies some corporation that isn't actually established. Perhaps the wording should be updated to reflect the intent (as I see it), which is likely "copyrighted per commit by the given code author".

One reason for asking users to relinquish copyrights in FOSS projects is to prevent issues in the future where copyright is used to revoke authorization to use contributed code ([I remember this incident vividly](https://blog.jwf.io/2020/04/open-source-minecraft-bukkit-gpl/)). Also, relicensing may be necessary to have cocotb distributed as a part of other products under other licenses. Without relinquishing rights, we would have to ask every contributor, which isn't possible. But now that I have had my coffee, I realize this is not what the copyright line is actually implying. I'm just projecting how I've operated. I'm a volunteer. I'm working to make something else better. I do not care to have any claim. If one day I am kicked out and cocotb goes closed source and commercial, I'll just pick up doing open source work from HEAD.